src/wikibots/flickr.py

```
@@ -18,44 +27,287 @@ class FlickrBot(BaseBot):
+            'circa': WbTime.PRECISION['year'],
+        }
 
-        if not creator_claim.has_qualifier(WikidataProperty.Url, 'https://commons.wikimedia.org/wiki/User:Jmabel'):
-            _q = creator_claim.qualifiers.get(WikidataProperty.Url)
-            url_qualifier: Claim = _q[0] if _q else Claim(self.commons, WikidataProperty.Url)
-            url_qualifier.setTarget('https://commons.wikimedia.org/wiki/User:Jmabel')
+        if date_taken['granularity'] not in precision_map:
+            error(f'Unrecognised date granularity: {date_taken}')
+            return
+
+        ts = Timestamp.fromISOformat(date_taken['value'].isoformat())
```

Ruff (0.8.2)

Local variable `ts` is assigned to but never used

Remove assignment to unused variable `ts`

(F841)